packages/web/src/components/apps/diff-modal.tsx

```diff
@@ -0,0 +1,82 @@
+import { cn } from '@/lib/utils';
+import { Button } from '@srcbook/components';
+import {
+  Dialog,
+  DialogContent,
+  DialogTitle,
+  DialogDescription,
+} from '@srcbook/components/src/components/ui/dialog';
+import { Undo2Icon, XIcon } from 'lucide-react';
+import type { FileDiffType } from './types';
+import { DiffSquares, DiffStats } from './diff-stats';
+import { DiffEditor } from './editor';
+
+type PropsType = {
+  onUndoAll: () => void;
+  onClose: () => void;
+  files: FileDiffType[];
+};
+
+export default function DiffModal({ files, onClose, onUndoAll }: PropsType) {
+  return (
+    <Dialog
+      open
+      onOpenChange={(open) => {
+        if (open === false) {
+          onClose();
+        }
+      }}
+    >
+      <DialogContent
+        className={cn('w-[95vw] h-[95vh] max-w-none p-0 gap-0 flex flex-col')}
+        hideClose
+      >
+        {/* Got browser console warnings without this */}
```

Comment:
I have given up on using the `DialogDescription` and `DialogHeader` and such because of these annoying warnings. They're setup as `<p>` which creates headaches

Reply:
They emit browser errors/warnings if not there.